src/main/kotlin/no/nav/familie/ba/sak/nare/Resultat.kt

```diff
@@ -1,28 +1,56 @@
 package no.nav.familie.ba.sak.nare
 
 enum class Resultat {
-    JA {
+    OPPFYLT {
 
         override infix fun og(other: Resultat): Resultat = other
-        override infix fun eller(other: Resultat): Resultat = JA
-        override fun ikke(): Resultat = NEI
+        override infix fun eller(other: Resultat): Resultat = OPPFYLT
+        override fun ikke(): Resultat = IKKE_OPPFYLT
     },
 
-    NEI {
+    IKKE_OPPFYLT {
 
-        override infix fun og(other: Resultat): Resultat = NEI
+        override infix fun og(other: Resultat): Resultat = IKKE_OPPFYLT
         override infix fun eller(other: Resultat): Resultat = other
-        override fun ikke(): Resultat = JA
+        override fun ikke(): Resultat = OPPFYLT
     },
 
-    KANSKJE {
+    IKKE_VURDERT {
 
-        override infix fun og(other: Resultat): Resultat = if (other == JA) KANSKJE else other
-        override infix fun eller(other: Resultat): Resultat = if (other == NEI) KANSKJE else other
-        override fun ikke(): Resultat = KANSKJE
+        override infix fun og(other: Resultat): Resultat = if (other == OPPFYLT) IKKE_VURDERT else other
+        override infix fun eller(other: Resultat): Resultat = if (other == IKKE_OPPFYLT) IKKE_VURDERT else other
+        override fun ikke(): Resultat = IKKE_VURDERT
     };
 
     abstract infix fun og(other: Resultat): Resultat
     abstract infix fun eller(other: Resultat): Resultat
     abstract fun ikke(): Resultat
+}
+
+enum class RestResultat {
+    OPPFYLT,
+    IKKE_OPPFYLT,
+    IKKE_VURDERT,
+    JA,
+    NEI,
+    KANSKJE
+}
+
+fun Resultat.tilRestResultat(): RestResultat {
+    return when(this) {
+        Resultat.OPPFYLT -> RestResultat.JA
+        Resultat.IKKE_OPPFYLT -> RestResultat.NEI
+        Resultat.IKKE_VURDERT -> RestResultat.KANSKJE
+    }
+}
+
+fun RestResultat.tilResultat(): Resultat {
+    return when(this) {
+        RestResultat.OPPFYLT -> Resultat.OPPFYLT
+        RestResultat.JA -> Resultat.OPPFYLT
```

Comment:
```suggestion
        RestResultat.OPPFYLT, RestResultat.JA -> Resultat.OPPFYLT
```

kan man forenkle litt ved å slå sammen? Bli da også tydelig at det er tre resultater som det (midlertidig) finnes to input-muligheter for

Reply:
Ajaj, viste ikke at dette var mulig - nice!